aip/general/0162.md

```diff
@@ -66,9 +66,9 @@ message BookRevision {
 - The `message` name **must** be named `{ResourceType}Revision`.
 - The resource revision **must** contain a field with a message type of the
   parent resource, with a field name of `snapshot`.
-    - The value of `snapshot` **must** be the configuration of the parent
+  - The value of `snapshot` **must** be the configuration of the parent
       at the point in time the revision was created.
-- The resource revision **must** contain a `create_time` field (see [AIP-142][]).
+- The resource revision **must** contain a `create_time` field (see (AIP-142)).
```

Comment:
We want this text to be linked so I think we want it to be `[AIP-142][]` still, but add the ref down at the bottom `[AIP-142]: ./0142.md`

I forget when it auto-links text like AIP-###, that might be a GitHub thing, not an AIP site thing but IDK.

Reply:
I took the brackets away and switched to parens because I saw that earlier in "The `message` **must** be annotated as a resource (AIP-123)." - I remember getting tripped up by this before and doing something that worked on GitHub but not the site

Reply:
I'm going to be honest, I don't *think* it consistently works, and the variety of linked v. unlinked AIP refs we have does not inspire confidence. I would rather just have the proper link we know works - unless you've rendered this locally and can verify that it works without an explicit link?

Reply:
Nevermind I found the site-generator code for "hot linking" - this is fine: https://github.com/aip-dev/site-generator/blob/1d54709e6e6d39c35d20ac1008ddbe63b753d5b6/aip_site/models/aip.py#L59-L72